utils/sb2

```diff
@@ -168,8 +168,8 @@ function locate_target_nsswitch_conf()
 		-L $SBOX_LIBSB2 -- $SBOX_DIR/bin/sb2-show \
 		which /var/run/nscd/socket \
 		>$SBOX_SESSION_DIR/path_to_nscd_socket.conf
-	NSSWITCH_CONF_PATH=`cat $SBOX_SESSION_DIR/path_to_nsswitch.conf`
-	NSCD_SOCKET_PATH=`cat $SBOX_SESSION_DIR/path_to_nscd_socket.conf`
+    read -r NSSWITCH_CONF_PATH < $SBOX_SESSION_DIR/path_to_nsswitch.conf
+    read -r NSCD_SOCKET_PATH < $SBOX_SESSION_DIR/path_to_nscd_socket.conf
```

Comment:
Indentation mismatch, also some other places have similar, recheck the indentations in your changes.